### PR TITLE
Reorganize user paths

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | none |
| Versions      | 4.5 and up |
| Edition       | Commerce |

Reorganize information in the store manager's path

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages

### CODE REVIEW

**Comment by dabrt** [comment]
@radzyu Please approve the changes.

**Comment by radzyu** [comment]
One general remark related to intro text:
- Maintain **product** pricing and shipping cost information
Instead of this point (related to content editor)
- Create engaging marketing content and publish it on the website
please add:
- Manage and track orders
- Manage shipping and payments methods available in your store

**Review by radzyu — COMMENTED** [review]
Some small remarks related to intro text

**Review by radzyu — APPROVED** [review]

File: docs/persona_paths/administrator.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Application administrator role
+
+When you perform administrative duties for your organization, you often are responsible for the configuration and maintenance of [[= product_name =]]-based application, as well as solve architectural challenges and performance problems.
+
+In [[= product_name =]] universe, most of such tasks are done outside of the UI.
```
Comment (MagdalenaZuba) [inline docs/persona_paths/administrator.md:5]: In [[= product_name =]] universe, -> In [[= product_name =]],

File: docs/pim/manage_prices.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+description: Products can have a base price and custom prices defined per custom group and per currency.
+---
+
+# Work with prices
+
+[Products in catalog](products.md#products) can have a base price and custom prices defined per [customer group](../customer_management/manage_customers.md) and per [currency](../pim/manage_currencies.md).
+
+With price management features, you can set up product prices manually.
```
[the remark this answers is not in the document]
Comment (radzyu) [reply docs/pim/manage_prices.md:9]: maybe prices are automatic when you use global and custom price rules :)

[the remark this answers is not in the document]
Comment (dabrt) [reply docs/pim/manage_prices.md:9]: This is old content, I was merely renaming a file. I'll remove the "can" to eliminate the suggestion that another possibility exists.

**Pull request merged**